scripts/gitlab/bulld_and_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I'm guessing this file was created for your own testing purposes and won't be included in the final PR?

Reply:
I think this script is used by the GitLab jobs...

Reply:
There is another file scripts/gitlab/build_and_test.sh that is very similar, which I assumed was the actual GitLab script.

Reply:
Whoops, I misread this filename.

Reply:
Good catch !
It copy created by mistake (typo). I thought I deleted it, but I may have resurrected it while cleaning the branch.
Corrected.
Thanks